public/fonts/Inter/Fonts.ts

```diff
@@ -0,0 +1,23 @@
+import * as InterRegular from "../fonts/Inter/Inter-Regular.woff2";
```

Comment:
Are you reading my comments?????

Reply:
also for what you need all fonts? Are you using font-weight 100,200,300 in project?